packages/apps/src/NodeInfo.tsx

```diff
@@ -16,8 +16,15 @@ import translate from './translate';
 
 type Props = I18nProps & {};
 
+export function isTestChain(chain?: string): boolean {
```

Comment:
Generally prefer that re-usable functions gets split - put it in a separate file. (We probably need to think about moving this into common/api at some point, it is a general utility after all - especially now that is is cleaner)

Reply:
agreed. moved to ui-react-rx/src/util as discussed